security/container_security/security-platform.adoc

```diff
@@ -0,0 +1,46 @@
+[id="security-platform"]
+= Securing the container platform
+include::modules/common-attributes.adoc[]
+:context: security-platform
+
+toc::[]
+
+{product-title} and Kubernetes APIs are key to automating container management at scale. APIs are used to:
+
+- Validate and configure the data for pods, services, and replication controllers.
+- Perform project validation on incoming requests and invoke triggers on other
+major system components.
+
+Security-related features in {product-title} that are based on Kubernetes include:
+
+* Multitenancy, which combines Role-Based Access Controls and network policies
+to isolate containers at multiple levels.
+* Admission plug-ins, which form boundaries between an API and those
+making requests to the API.
+
+{product-title} uses Operators to automate and simplify the management of
+Kubernetes-level security features.
+
+// Multitenancy
+include::modules/security-platform-multi-tenancy.adoc[leveloffset=+1]
+
+// Admission plug-ins
+include::modules/security-platform-admission.adoc[leveloffset=+1]
+
+// Authentication and authorization
+include::modules/security-platform-authentication.adoc[leveloffset=+1]
+
+// Managing certificates for the platform
+include::modules/security-platform-certificates.adoc[leveloffset=+1]
+
+.Additional resources
+* xref:../../welcome/oke_about.adoc#oke-about[About {product-title}]
```

Comment:
So no, this OCE (renamed OKE) is actually different from OCP, a smaller piece of it I believe. I think this is a poor place for the docs on it, because it's right where you'd expect a section on "About OCP" to be, in the OCP docs.

Also, this link doesn't work for the OKD docs anyways.

So, instead of pointing them to this section, I'd consider linking to https://security-bz1756602--ocpdocs.netlify.app/openshift-enterprise/latest/architecture/architecture.html#architecture-platform-introduction_architecture

Reply:
Done.